### PR TITLE
Main page demo

### PR DESCRIPTION
**Pull request by Fusionqq**
depends on #5

### CODE REVIEW

**Comment by Drapegnik** [comment]
`Sashkina_vetka_za_css_i_dvor_strelyau_v_ypor` 🤦‍♂️

**Comment by Drapegnik** [comment]
`npm run lint`:
```
Message:
    Failed with 147 errors
```

please correct mistakes, if some rules from lint are bad, we can discuss and switch-off it

File: src/main-page/main-page.html [hunk 1]
```diff
@@ -3,50 +3,56 @@
 <head>
 	<meta charset="UTF-8">
 	<title>Main page</title>
+	<link rel="stylesheet" href="../../node_modules/font-awesome/css/font-awesome.min.css">
+	<!--<link rel="stylesheet" href="../../node_modules/bulma/css/bulma.css">-->
```
Comment (Drapegnik) [inline src/main-page/main-page.html]: why you did this??

File: src/main-page/base.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    line-height: 1.5;
+    font-family: BlinkMacSystemFont, -apple-system, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", "Helvetica", "Arial", sans-serif;
+}
+
+body, html {
+    height: 100%;   
+}
+
+p {
+    margin: 0px;
```
Comment (tyndria) [inline src/main-page/base.scss]: I prefer use just margin: 0, specifying any units in a case of zero value is redundant, probably

**Comment by Drapegnik** [comment]
screen shot 2017-07-08 at 15 12 29
screen shot 2017-07-08 at 15 12 59
screen shot 2017-07-08 at 15 13 07

File: src/main-page/first-block.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        flex-grow: 1;
+        display: flex;
+        flex-direction: column;
+    }
+
+    &__left-side-innerOne {
+        flex-grow: 1;
+        background-color: rgb(0, 209, 177);
+        border-radius: 5px;
+        padding: 15px;
```
Comment (Drapegnik) [inline src/main-page/first-block.scss]: I think `px` good in height, margins, paddings and borders property. But bad in weight and other.

**Comment by Drapegnik** [comment]
.

File: src/main-page/first-block.scss (hunk 3 above)
Comment (Drapegnik) [inline src/main-page/first-block.scss]: dsfsd

Comment (Drapegnik) [inline src/main-page/first-block.scss]: https://learn.javascript.ru/css-units

Comment (Drapegnik) [inline src/main-page/first-block.scss]: https://learn.javascript.ru/css-units

**Comment by Drapegnik** [comment]
[main page markup](https://trello.com/c/fQsV0aix/11-main-page-markup)

**Pull request closed without merging**